lib/packExternalModules.js

```diff
@@ -9,7 +9,7 @@ const isBuiltinModule = require('is-builtin-module');
 const Packagers = require('./packagers');
 
 function rebaseFileReferences(pathToPackageRoot, moduleVersion) {
-  if (/^(?:file:[^/]{2}|\.\/|\.\.\/)/.test(moduleVersion)) {
+  if (/^(file:[^\/]|\.\/|\.\.\/)/.test(moduleVersion)) {
```

Comment:
You should keep the `?:` as it marks the group as non-capturing group here.

Reply:
Additionally, I do not know if a `file:` prefix is a valid file URL. Normally only `file://` is valid, but relative files, as in your example, should be written as `file://./` or without the file prefix?

Reply:
Sure thing, I can put the non-capturing group specifier back.

Regarding the prefix, this is the function that handles package resolution in `npm`: https://github.com/npm/npm-package-arg/blob/latest/npa.js#L51-L79. As you can see, the prefix it uses is simply `file:` rather than `file://`.

Reply:
In fact, a much more correct solution here would be to pull in `npm-package-arg` as a dependency, but I didn't want to add a new dependency on your behalf.

Reply:
@talbenari1 For now we can go with the light-weight regex approach. 

However, I like the idea going with `npm-package-arg` but think this would be more like a overhaul of different parts of the dependency management within the plugin - at least to make it consistent. If you want, you can open a new issue and put the use of the module under discussion there. I'm curious what others will think about the idea and we might get a broader picture and decide how to proceed.

Reply:
Agree with @HyperBrain, let's keep the regex approach for this PR, but look into using `npm-package-arg` going forward. I'm all for moving complexity out of the plugin 😄

Reply:
Has this comment been addressed?

Reply:
```suggestion
  if (/^(file:[^/]|\.\/|\.\.\/)/.test(moduleVersion)) {
```